geomesa-core/src/main/scala/org/locationtech/geomesa/core/data/AccumuloFeatureSource.scala

```diff
@@ -67,10 +67,11 @@ trait AccumuloAbstractFeatureSource extends AbstractFeatureSource with Logging w
     new AccumuloFeatureCollection(self, query)
   }
 
-  override def getFeatures(query: Query): SimpleFeatureCollection = tryLoggingFailures(getFeaturesNoCache(query))
+  override def getFeatures(query: Query): SimpleFeatureCollection =
+    tryLoggingFailures(getFeaturesNoCache(query))
 
   override def getFeatures(filter: Filter): SimpleFeatureCollection =
-    tryLoggingFailures(getFeatures(new Query(getSchema().getTypeName, filter)))
+    getFeatures(new Query(getSchema().getTypeName, filter))
```

Comment:
Didn't like the tryLoggingFailures?  

We added this to try and log underlying errors (which GT otherwise swallows).